package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@storyous/common-utils",
-  "version": "16.4.1",
+  "version": "16.4.2",
```

Comment:
We are adding new feature, in such case we should increase the middle number. The last number is only for fixes.

Reply:
+After you change version in package.json then run `npm install`, it will update `package-lock.json` with this new version as well. When you not do it, it can lead to conflicts in future updates.